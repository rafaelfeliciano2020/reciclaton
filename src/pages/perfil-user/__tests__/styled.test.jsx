import React from 'react';
import {
  StyledPerfilDiv,
  StyledImgRankCenter,
  StyledUserImage,
  StyledRankUser,
  InfoDiv,
  StyledReportDiv,
  ReportButton,
  FuncButton,
  StyledPerfilMaterials,
  StyledMaterials

} from "../styled";
import renderer from 'react-test-renderer';

it("render styledPerfilDiv", () => {
  const tree = renderer.create(<styledPerfilDiv />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("render StyledImgRankCenter", () => {
  const tree = renderer.create(<StyledImgRankCenter />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("render StyledUserImage", () => {
  const tree = renderer.create(<StyledUserImage />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("render StyledRankUser", () => {
  const tree = renderer.create(<StyledRankUser />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("render InfoDiv", () => {
  const tree = renderer.create(<InfoDiv />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("render StyledReportDiv", () => {
  const tree = renderer.create(<StyledReportDiv />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("render ReportButton", () => {
  const tree = renderer.create(<ReportButton />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("render FuncButton", () => {
  const tree = renderer.create(<FuncButton />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("render StyledPerfilMaterials", () => {
  const tree = renderer.create(<StyledPerfilMaterials />).toJSON();
  expect(tree).toMatchSnapshot();
})

it("render StyledMaterials", () => {
  const tree = renderer.create(<StyledMaterials />).toJSON();
  expect(tree).toMatchSnapshot();
})