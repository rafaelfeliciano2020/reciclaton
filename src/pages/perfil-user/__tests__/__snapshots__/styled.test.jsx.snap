// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render FuncButton 1`] = `
<button
  className="sc-fujyUd sc-pNWxx kWuwGs hgVpPr"
/>
`;

exports[`render InfoDiv 1`] = `
<div
  className="sc-gKAblj feBrzO"
/>
`;

exports[`render ReportButton 1`] = `
<button
  className="sc-fujyUd kWuwGs"
/>
`;

exports[`render StyledImgRankCenter 1`] = `
<div
  className="sc-jSFkmK bXBWRb"
/>
`;

exports[`render StyledMaterials 1`] = `
<div
  className="sc-dlnjPT qJemq"
/>
`;

exports[`render StyledPerfilMaterials 1`] = `
<div
  className="sc-gtssRu duoymd"
/>
`;

exports[`render StyledRankUser 1`] = `
<div
  className="sc-eCApGN iWBcgg"
/>
`;

exports[`render StyledReportDiv 1`] = `
<div
  className="sc-iCoHVE kcubFd"
/>
`;

exports[`render StyledUserImage 1`] = `
<img
  className="sc-hKFyIo fxFGM"
/>
`;

exports[`render styledPerfilDiv 1`] = `<styledPerfilDiv />`;
